docs/source/delta-uniform.md

```diff
@@ -16,42 +16,42 @@ To enable UniForm, you must fulfill the following requirements:
 
 - The table must have column mapping enabled. See [_](delta-column-mapping.md).
 - The Delta table must have a `minReaderVersion` >= 2 and `minWriterVersion` >= 7.
-- Writes to the table must use <Delta> 3.0 or above.
+- Writes to the table must use <Delta> 3.1 or above.
```

Comment:
This makes it look like Uniform is available only from the 3.1. May be create a table with each Iceberg Compat version and Delta Lake version and the feature availability? One [example](https://github.com/delta-io/delta/commit/ba758a6660d727f5aaa4c7262292e675bca1753d).

Currently, we have a doc tree for each new version of Delta Lake, but we have plans to move one doc tree for all versions.

Reply:
we want to explicitly discourage IcebergCompatV1 or what uniform is in delta 3.0 because v2 is strictly better and has less restrictions. User can always refer to delta protocol for IcebergCompatV1 but they should just not use it.